File: wk1-c/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do{
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);

    // set loop for the rows of both pyramids (building from top to bottom)
    for (int row = 0 ; row < height ; row++){
        // set loop for columns of first pyramid
        for (int colOne = 0 ; colOne < height; colOne++){
            if (row + colOne >= height - 1){
                printf("#");
            }
            else printf(" ");
        }
        printf("  ");
        // set loop for columns of second pyramid
        for (int colTwo = height - 1; colTwo > -1; colTwo--){
            if (row + colTwo >= height - 1){
                printf("#");
            }
        }
        printf("\n");
    }
}

File: wk6-python/mario.py
def main():
    # Get user input for the height of pyramid
    height = get_height()
    # Set loop for rows of both pyramids (building from top to bottom)
    for row in range(height):
        # Set loop for columns of first pyramid
        for colOne in range(height):
            if row + colOne >= height - 1:
                print('#', end="")
            else:
                print(' ', end="")
        print('  ', end="")
        # Set loop for columns of second pyramid
        # Start at height - 1, go to 0 inclusive, decrement by 1
        for colTwo in range(height - 1, -1, -1):
            if row + colTwo >= height - 1:
                print('#', end="")
        print('')


def get_height():
    while True:
        try:
            height = int(input("Height: "))
            if height > 0 and height < 9:
                return height
            else:
                print("Pick a height between 1 and 8.")
        except ValueError:
            print("Please type an integer.")


main()